src/main/java/org/sagebionetworks/template/repo/cloudwatchlogs/CloudwatchLogsConfigValidator.java

```diff
@@ -21,11 +23,17 @@ public CloudwatchLogsConfig validate() {
         validateEnvironments(logDescriptors);
         for (EnvironmentType k: logDescriptors.keySet()) {
             List<LogDescriptor> descriptors = logDescriptors.get(k);
-            if (descriptors.size() != 3) {
-                throw new IllegalStateException("Each environment should have 3 logGroups");
+            if (descriptors.size() != 2) {
+                throw new IllegalStateException("Each environment should have 2 logGroups.");
             }
+            List<LogType> types = new ArrayList<>();
             for (LogDescriptor d: descriptors) {
                 this.validateLogDescriptor(d);
+                if (types.contains(d.getLogType())) {
```

Comment:
You can check the return type of the add method (false if it was not added because it's already in the collection)